src/power_grid_model/validation/validation.py

```diff
@@ -221,6 +221,28 @@ def validate_ids_exist(update_data: Dict[str, np.ndarray], input_data: SingleDat
     return list(chain(*errors))
 
 
+def process_power_sigma_and_p_q_sigma(data: SingleDataset, sensor: str) -> None:
+    """
+    Helper function to process the required list when both `p_sigma` and `q_sigma` exist
+    and valid but `power_sigma` is missing. The field `power_sigma` is set to 1 in this case.
```

Comment:
i think this comment is out of date, right?

Reply:
Indeed, "...set to the norm of..."